### PR TITLE
feature/CEMCA-15: Agregue componente Header

### PR DESCRIPTION
**Pull request by samuelching-sealabs**
Hay componente header para futuras paginas, ademas, se agrego pagina nueva llamada Seminars.

### CODE REVIEW

File: src/components/PageHeader/index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import { Box } from '@mui/material';
+import clsx from 'clsx';
+import fontsStyles from '../../styles/fontStyles';
+import useStyles from './styles';
+
+
+
+
+
+
```
Comment (jeanvega-sealabs) [inline src/components/PageHeader/index.js:11]: Remove whitte spaces

**Review by jeanvega-sealabs — CHANGES_REQUESTED** [review]

File: src/components/PageHeader/index.js (hunk 1 above)
Comment (samuelching-sealabs) [reply src/components/PageHeader/index.js]: White blanks were removed

File: src/components/PageHeader/index.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        <Box className={classes.HeadWrapper}>
+          <Box className={classes.HeaderContentWrapper}>
+            <Box>
+              <h1 className={fontClasses.h1}>{title}</h1>
+              <p className={clsx(fontClasses.p, classes.headerSubtitle)}>{description}</p>
+            </Box>
+          </Box>
+        </Box>
+      );    
+
```
Comment (axelmairena-sealabs) [inline src/components/PageHeader/index.js:21]: Eliminar espaciio

File: src/components/PageHeader/styles.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        width:'70%',
+        color: "white",
+        marginTop: "20px",
+        [themeBreakpoints.down('sm')]: {
+            width:'100%',
+        },
+    }
+}))
+
+export default useStyles
```
Comment (axelmairena-sealabs) [inline src/components/PageHeader/styles.js:47]: Agregar newline al end del file

File: src/pages/Seminars/index.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+const Seminars =()=>{
+    return (
+      <Box>
+        <PageHeader title="Fomentemos el conocimiento" description="Con este espacio buscamos capacitar mujeres en el fortalecimiento de sus capacidades y competencias en gestión empresarial, abocándonos a la creación de proyectos que impulsaran un modelo que desde nuestra organización contempla un crecimiento de las mujeres dentro de sus comunidades vinculándolas al desarrollo socioeconómico del mismo."/>
+      </Box>
+    );
+}
+
+export default Seminars;
```
Comment (axelmairena-sealabs) [inline src/pages/Seminars/index.js:13]: agregar new line al end del file

File: src/components/PageHeader/index.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+          <Box className={classes.HeaderContentWrapper}>
+            <Box>
+              <h1 className={fontClasses.h1}>{title}</h1>
+              <p className={clsx(fontClasses.p, classes.headerSubtitle)}>{description}</p>
+            </Box>
+          </Box>
+        </Box>
+      );    
+  
+  }
```
Comment (axelmairena-sealabs) [inline src/components/PageHeader/index.js:22]: agregar new line

**Review by axelmairena-sealabs — APPROVED** [review]

**Review by axelmairena-sealabs — CHANGES_REQUESTED** [review]
Revisar comentarios anteriores

File: src/components/PageHeader/index.js (hunk 2 above)
Comment (samuelching-sealabs) [reply src/components/PageHeader/index.js]: Listo

**Pull request merged**